Make knative build a module (#4606)
kyma-project/kyma: resources/knative-serving-crds/README.md

# Knative Serving CRDs

## Overview

> This module is still experimental.

Reviewer: ```suggestion
>**NOTE**: This is an experimental module.
```

> This chart includes [knative-serving-crds](https://github.com/knative/docs/tree/master/docs/serving) release files.

Reviewer: ```suggestion

This chart includes [knative-serving-crds](https://github.com/knative/docs/tree/master/docs/serving) release files.
```
Add a line, otherwise this part is included in the note.

> Install CRDs before installing the actual resources as they depend on one another. 

Reviewer: ```suggestion
Install CRDs before installing the actual resources as they depend on each other. 
```

> We need to install crds before installing the actual resources since its a dependency for the resources being installed

Reviewer: ```suggestion
Install CRDs before installing the actual resources as they depend on one another. 
```
Same question as above.

Author: We want to be sure that crds are installed before the resources are installed. If we have them in same file then we cannot gaurantee it.

Author: fixed
